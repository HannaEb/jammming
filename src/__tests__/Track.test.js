import Track from '../Components/Track/Track.js';
import renderer from 'react-test-renderer';

describe('Track', () => {
    it('matches the snapshot', () => {
        const track = {name: 'Name', artist: 'Artist', album: 'Album'}
        const tree = renderer.create(<Track track={track} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
