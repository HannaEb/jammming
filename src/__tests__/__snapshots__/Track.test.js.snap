// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Track matches the snapshot 1`] = `
<div
  className="Track"
>
  <div
    className="Track-information"
  >
    <h3>
      Name
    </h3>
    <p>
      Artist
       | 
      Album
    </p>
  </div>
  <button
    className="Track-action"
    onClick={[Function]}
  >
    +
  </button>
</div>
`;
